pkg/common/flags/general.go

```diff
@@ -0,0 +1,71 @@
+package flags
+
+import "github.com/urfave/cli/v2"
+
+var (
+	NetworkFlag = cli.StringFlag{
+		Name:    "network",
+		Aliases: []string{"n"},
+		Usage:   "Network to use. Currently supports 'preprod', 'holesky' and 'mainnet'",
```

Comment:
Networks should be either `holesky` or `mainnet`. You'll need an `environment` flag that could have: `preprod`, `testnet` or `prod`

Reply:
fair point.

Reply:
updated 0aeb8f1a8894dac95c5306f67f45187b8d4b0e95